src/features/list-view/styles/ListView.scss

```diff
@@ -1,14 +1,16 @@
 @import '../../../styles/variables';
 
-.metadata-views-list-view {
+$prefix: bdl-MetadataViews-ListView;
```

Comment:
This will be defined as a global variable. Is that intended?

Reply:
Good catch. I can revise to...

```.bdl-MetadataViews-ListView {
    $prefix: bdl-MetadataViews-ListView;
   ...
```

Thoughts?

Reply:
I'm not sure I see the value of the variable. Part of the benefit of SUIT is to avoid nesting and increases in specificity. The code duplication without the variable is minimal, is it not?

Reply:
There's this -> .bdl-icon-sort-chevron.bdl-MetadataViews-ListView-isSortAsc which I find barely legible.

As it stands today, I find repetitiveness of the prefix to be an eyesore and this is exacerbated as the scss file will only grow over time. 

Given the lengthiness of this prefix, I prefer DRY over other virtues.

Reply:
I agree, that class name isn't acceptable, but the issue isn't caused by SUIT or the conventions I describe; it's caused by the component architecture. Each component should be a separate concern (and file, preferably) with a class to match.

Example: `.bdl-MetadataList-sort---asc`

Reply:
I take it back. If searchability is important, then let's do that instead of DRY.